src/coreclr/src/tools/Common/System/NotImplemented.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think this file is not used either (just included from the csproj for no reason).